### PR TITLE
Add includeInvalidTxs parameter into bitcore-client & BWS query methods

### PR DESCRIPTION
**Pull request by clabby**
#3227

### CODE REVIEW

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts [hunk 1]
```diff
@@ -288,7 +288,7 @@ export class V8 {
       .catch(cb);
   }
 
-  getTransactions(wallet, startBlock, cb) {
+  getTransactions(wallet, startBlock, cb, includeInvalidTxs = false) {
```
Comment (kajoseph) [inline packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts:291]: Let's keep the callback as the last param

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8/client.ts [hunk 2]
```diff
@@ -126,25 +127,24 @@ export class Client {
...
+    const rawQuery = {
+      startBlock,
+      endBlock,
+      startDate,
+      endDate,
+      tokenAddress,
+      includeMempool: includeMempool && true,
+      includeInvalidTxs
+    };
+    const query = _.map(_.keys(rawQuery), key => rawQuery[key] && `${key}=${rawQuery[key]}`).join('&');
```
Comment (kajoseph) [inline packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8/client.ts:144]: We shouldn't need lodash here

**Review by kajoseph — COMMENTED** [review]
Overall looks good, just a couple of minor changes

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts (hunk 1 above)
Comment (clabby) [reply packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts]: Updated

File: packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8/client.ts (hunk 2 above)
Comment (clabby) [reply packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8/client.ts]: Updated

**Pull request closed without merging**